code/game/machinery/rechargestation.dm

```diff
@@ -111,20 +110,24 @@
 /obj/machinery/recharge_station/proc/toggle_restock(mob/user)
 	if(sendmats)
 		sendmats = FALSE
-		balloon_alert(user, "restocking from ore silo: disabled")
+		if(user)
+			balloon_alert(user, "restocking from ore silo: disabled")
```

Comment:
We talked about the balloon alerts a little but i think they outta be shortened or swapped for something like say(), we can't expect people to read the whole thing without spamming it

Reply:
``silo link: disabled/enabled`` looks short enough.

Reply:
Replaced with say() for these four. The restocking messages of `+ 15000u iron` are still balloon alerts.